docs/recipes/install/rocky9/x86_64/confluent/slurm/steps.tex

```diff
@@ -156,7 +156,7 @@ \subsection{Enable \OHPC{} repository for local use} \label{sec:enable_repo}
 
 % begin_ohpc_run
 \begin{lstlisting}[language=bash,keywords={},basicstyle=\fontencoding{T1}\fontsize{8.0}{10}\ttfamily,literate={ARCH}{\arch{}}1 {-}{-}1]
-[sms](*\#*)  (*\install*) epel-release
+[sms](*\#*) (*\install*) epel-release
```

Comment:
The recommendation when installing `epel-release` it to do `/usr/bin/crb enable` as a second command. My recommendation would be to install `epel-release` on the SMS and on the compute nodes as well as run ` /usr/bin/crb enable` on the SMS and compute nodes.